planning/autoware_behavior_velocity_template_module/src/manager.hpp

```diff
@@ -37,7 +37,7 @@ namespace behavior_velocity_planner
  *
  * @param node A reference to the ROS node.
  */
-class TemplateModuleManager : public SceneModuleManagerInterface
+class TemplateModuleManager : public ::behavior_velocity_planner::SceneModuleManagerInterface
```

Comment:
When [behavior_velocity_planner_common/scene_module_interface.hpp/SceneModuleInterface](https://github.com/autowarefoundation/autoware.universe/blob/010965b9b1f8ba8056fdbe5e6a2aa8eb3e68e6a0/planning/behavior_velocity_planner_common/include/behavior_velocity_planner_common/scene_module_interface.hpp#L76) is moved under `autoware` namespace, this will need to be updated too right?

Just asking out of curiosity, right now it works and links correctly.

Reply:
Yes, I did it this way so that this package will fail when I refactor the other ones